openlabs_report_webkit/__init__.py

```diff
@@ -29,56 +24,45 @@
 from trytond.tools import file_open
 from trytond.pool import Pool
 from trytond.transaction import Transaction
-from trytond.report import Report, TranslateFactory, Translator, FORMAT2EXT
+from trytond.report import Report, TranslateFactory, Translator
 from executor import execute
 
 
 class ReportWebkit(Report):
+    render_method = "webkit"
```

Comment:
I think we could use config for this. For the moment, we could squash and merge this patch.